draft-ietf-ppm-dap.md

```diff
@@ -2680,35 +2542,53 @@ service has no obvious way to authenticate its report uploads.
 ## Anonymizing proxies {#anon-proxy}
 
 Client reports can contain auxiliary information such as source IP, HTTP user
-agent or in deployments which use it, Client authentication information, which
-could be used by Aggregators to identify participating Clients or permit some
-attacks on robustness. This auxiliary information could be removed by having
-Clients submit reports to an anonymizing proxy server which would then use
-Oblivious HTTP {{!I-D.draft-ietf-ohai-ohttp-08}} to forward reports to the
-DAP Leader, without requiring any server participating in DAP to be aware of
-whatever Client authentication or attestation scheme is in use.
+agent, or Client authentication information (in deployments which use it, see
+{{client-auth}}). This metadata can be used by Aggregators to identify
+participating Clients or permit some attacks on robustness. This auxiliary
+information can be removed by having Clients submit reports to an anonymizing
+proxy server which would then use Oblivious HTTP
+{{!I-D.draft-ietf-ohai-ohttp-08}} to forward reports to the DAP Leader. In this
+scenario, Client authentication would be performed by the proxy rather than any
+of the participants in the DAP protocol.
+
+## Differential privacy {#dp}
+
+DAP deployments can choose to ensure their aggregate results achieve
+differential privacy [Vad16]. A simple approach would require the Aggregators to
+add two-sided noise (e.g. sampled from a two-sided geometric distribution) to
+aggregate shares. Since each Aggregator is adding noise independently, privacy
+can be guaranteed even if all but one of the Aggregators is malicious.
+Differential privacy is a strong privacy definition, and protects users in
```

Comment:
Is this sentence accurate/not overstating things? If I know a single measurement-with-noise value, and the distribution used for noise, I can determine a probability distribution for the measurement-without-noise value. My understanding is that noise values are typically "small", so this distribution would determine a small likely range for the true measurement value.

Reply:
IIUC, I think what I say is true for "Client DP" (noise added by client to original measurement), but not "Aggregator DP" (noise added by aggregator to aggregate share), looking to https://www.ietf.org/archive/id/draft-wang-ppm-differential-privacy-00.html.